app/controllers/switcher_controller.rb

```diff
@@ -0,0 +1,14 @@
+class SwitcherController < ApplicationController
+  before_action redirect_to root_path if Rails.env.production?
+
+  def cycles; end
+
+  def update
+    new_cycle = params[:change_cycle][:cycle_schedule_name]
+    SiteSetting.set(name: 'cycle_schedule', value: new_cycle)
+    Rails.application.reload_routes!
```

Comment:
So i'm kind of split on this. We definitely could just update the application controller to redirect to the correct page based on the CycleTimetable

i.e. when find is closed always redirect to the find is closed page, otherwise if you hit the find is closed page then redirect to the root path.

Interested to hear what other people think.

Reply:
I think if the ApplicationController option is simpler and both suits the current use case and is easy to extend (if / when required) then that may be the way to go. Either way might be good to have some docs